app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -150,14 +152,12 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
         ) as FormStep[]);
   const reviewAfterFailedSteps = [reviewFormStep] as FormStep[];
   const reviewWithInPersonSteps = reviewAfterFailedSteps.concat(inPersonSteps);
-  const afterSubmissionErrorSteps = docAuthSeparatePagesEnabled
-    ? reviewAfterFailedSteps
-    : reviewWithInPersonSteps;
+  const afterSubmissionErrorSteps =
+    docAuthSeparatePagesEnabled && isSelfieCaptureEnabled && !isInPersonStepEnabled
+      ? reviewAfterFailedSteps
+      : reviewWithInPersonSteps;
```

Comment:
IPP should always be included in the steps when there is a submission error.
I had to do some refactoring to uncover this. I was able to simplify this code as shown below. can you please simplify 151 - 161 by replacing it with following (or something to the likes)?  I thinks helps the code much more readable

```
  let steps = documentsFormSteps;
  if (isInPersonStepEnabled) {
    steps = inPersonSteps;
  } else if (submissionError) {
    steps = ([reviewFormStep] as FormStep[]).concat(inPersonSteps);
  }
```